### ISSUE TITLE
Informative Techniques pages are loaded with normative RFC-2119 terms like MUST

### ISSUE THREAD

**Issue by cookiecrook** [w3c/wcag#1150]
Informative Techniques pages are loaded with normative RFC-2119 terms like MUST. 

In Issue w3c/wai-wcag-supporting-documents-redesign#4, I mentioned that the techniques pages don't list that they are informative. The confusion is further compounded by many techniques pages using Normative (RFC-2119) terms like "must." I have not performed a full audit, but here's one example that uses the term "must" six times.

https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html

In a meeting at TPAC 2019 in Fukuoka, then Chair @awkawk acknowledged this as a problem and committed to getting it resolved.

I'd recommending the following: 
- rid the techniques documents of all instances of the term "must"
- review every instance of the word "should" to ensure it's clear the advice listed is one of many potential solutions, rather the only solution. 

I have anecdotal evidence that both lay readers and expert readers are confused by some of the usage of these terms in techniques documents.

I acknowledge that some concepts would be difficult to convey if we rid the documents of the word "should"—it's often the best term—but I think it's reasonable to remove the word "must" from non-normative documents, and review the usage of the other terms that may be misconstrued...

**Comment by hidde** [comment]
Just want to +1 this point, I have similar anecdotal evidence that support that the use of “must” in these documents can confuse some readers.

Have moved this issue from the [wai-wcag-supporting-documents-redesign](https://github.com/w3c/wai-wcag-supporting-documents-redesign/issues) repository, as content changes are explicitly [out of scope](https://github.com/w3c/wai-wcag-supporting-documents-redesign/wiki/Requirements-analysis#exclusions) for that particular project.

**Comment by patrickhlauke** [comment]
+1 here from me as well, and same about anecdotal evidence that authors (and even some auditors) misunderstand what techniques are trying to show (and even further, the subtle difference between sufficient and advisory techniques as well, or that failure techniques always fail but only if there is nothing else that may still satisfy the core ask of an SC in other ways)

**Comment by goodwitch** [comment]
+1  I have solid evidence that the techniques are considered "gospel" by
most people who have self-taught themselves a11y (or really, anyone who is
not deeply involved with the W3C).

I've begged for this problem to be resolved for years.  This is a root
cause of what I refer to as the "A11Y Wars
<https://docs.google.com/document/d/1L9OYaP052j2uxVu-8IXGxzwQsHdx8G6YKrxpHo7BZ_w/edit>".


*glenda sims* <glenda.sims@deque.com>, cpwa
<https://www.accessibilityassociation.org/certification>   | team a11y lead
| 512.963.3773

        deque systems <http://www.deque.com>  accessibility for good

**Comment by bruce-usab** [comment]
Wholly agree with avoiding <q>must</q> in the Understanding and Techniques docs.  In my experience it is pretty easy to rewrite these.  With this example:

> Therefore, functionality offered through motion **must** also be available by another mechanism. ... The user _must have_ the ability to turn off motion actuation to prevent such accidental triggering of functions.

Could be:

> Therefore, functionality offered through motion **needs to** also be available by another mechanism. ... The user **needs** the ability to turn off motion actuation to prevent such accidental triggering of functions.

@patrickhlauke wrote:

> failure techniques always fail but only if there is nothing else that may still satisfy the core ask of an SC in other ways

My understanding is that our failures are suppose be barriers?  Of course, there is usually the possibility of a Conforming Alternate Version.  Is that what you had in mind?

**Comment by patrickhlauke** [comment]
> My understanding is that our failures are suppose be barriers? Of course, there is usually the possibility of a Conforming Alternate Version. Is that what you had in mind?

it's this part that is often not clear to people. they often see this more as a "whenever this kind of thing is used, it's automatically a fail" without understanding the nuance that there might be some wider context/a way that even with that particular pattern that is a failure in isolation, something else may be (even on the same page) that still ensures that the same purpose/content is conveyed properly.

taking one trivial example ... somebody comes across https://www.w3.org/WAI/WCAG21/Techniques/failures/F3.html and thinks "so whenever a CSS background image is used to provide content, it's a fail" without understanding that it's quite possible to use CSS background images, provided there's also something else (e.g. visually hidden text, an `aria-label` somewhere, or similar) that conveys the same content. they're often seeing these as absolute edicts of "never use this particular technology", where really it's more "never use this completely on its own, in isolation, as the only way to do something"

**Comment by mraccess77** [comment]
Only tweak to @patrickhlauke comment is that the way F3 was written is that off-screen text won't pass it unless it is displayed when the CSS image is not displayed.  This technique was written this way to allow users with low vision to turn off and on background images as background images.  There is still a user need to be able to turn off decorative images but keep meaningful images.

**Comment by bruce-usab** [comment]
> taking one trivial example ... somebody comes across https://www.w3.org/WAI/WCAG21/Techniques/failures/F3.html and thinks ...

I think that might be _the_ archetype example!  So do what do you propose as a fix?

**Comment by alastc** [comment]
@cookiecrook can I just check - the document you pointed to is not a technique document, did you intend to cover the understanding documents as well?

Where an understanding document is making direct reference to the normative requirement, presumably the use of 'must' is ok there?

4 of the 6 uses of 'must' in that document are from the _normative_ glossary, which includes (in the literal server-side include sense) the relevant terms.

For the other two, I think the use of 'must' is ok if it is a direct reference to the normative requirement. For example, the first of those could be:
"Therefore, functionality offered through motion must also be available by another mechanism [ins]unless it meets one of the exceptions[/ins]."

In that context, it does seem to be an appropriate use of must.

**Comment by bruce-usab** [comment]
> It will then ask if you want to create a branch, and you can follow the journey from there.

@alastc I see you already did a [pull request](https://github.com/w3c/wcag/pull/1127/files/ada55ccfa42c25b599e7bb24c37c06e5471627a6..08d4f20e1f7c82d2cf022ab65933f7604f118cd0) for this edit.  Sorry to not be more timely!

**Comment by alastc** [comment]
Hi @bruce-usab - only on that particular SC, whilst I was getting it ready for CFC. There are plenty of others...

**Comment by patrickhlauke** [comment]
> > taking one trivial example ... somebody comes across https://www.w3.org/WAI/WCAG21/Techniques/failures/F3.html and thinks ...
> 
> I think that might be _the_ archetype example! So do what do you propose as a fix?

Coming in late, but ... here's what I'd propose https://github.com/w3c/wcag/issues/1567

**Comment by patrickhlauke** [comment]
I've had a cursory search over our current techniques and understanding documents. While the words "must" and "should" are used quite often, it seems to me (from a very high-level skim) that they are used in reference to the actual normative requirements they refer to - i.e. they don't introduce (or appear to introduce) any new normative requirements, but simply restate the normative requirements from WCAG's normative section.

Unless there's any specific examples (which we'd be happy to amend on a case-by-case basis - and incidentally, in all our work in the TF we've tried not to introduce/use any RFC-2119 terms ourselves) I'd close this issue for now.
